indico/util/spreadsheets.py

```diff
@@ -133,13 +137,13 @@ def send_csv(filename, headers, rows):
     return send_file(filename, buf, 'text/csv', inline=False)
 
 
-def send_xlsx(filename, headers, rows):
+def send_xlsx(filename, headers, rows, tz=None):
```

Comment:
If you add a new argument to a function with a docstring documenting the params, please add that argument in there too ;)